modules/atf/stdlib/argument_parser.lua

```diff
@@ -43,13 +43,13 @@ local function errmsg (msg)
   if msg:match ("%.$") == nil then msg = msg .. "." end
   print (prog .. ": error: " .. msg .. "\n")
   print (prog .. ": Try '" .. prog .. " --help' for help.")
-  quit(2)
+  quit(3)
```

Comment:
@VVeremjova , what changes? can you move this to some var? Or create some enum of consts with exit codes.